tests/st/test_ipip.py

```diff
@@ -29,3 +32,56 @@ class TestIPIP(TestBase):
     @skip("Not written yet")
     def test_ipip(self):
         pass
+
+    def test_ipip_addr_assigned(self):
+        with DockerHost('host', dind=False, start_calico=False) as host:
```

Comment:
Probably an overkill - but should we test it all is good when starting the node without any pool being added (and setting the node environment so that a pool is not added by the node command?)